shell/renderer/atom_renderer_client.cc

```diff
@@ -119,11 +119,12 @@ void AtomRendererClient::DidCreateScriptContext(
     node::tracing::TraceEventHelper::SetAgent(node::CreateAgent());
 
   // Setup node environment for each window.
-  v8::Local<v8::Context> context =
-      node::MaybeInitializeContext(renderer_context);
-  DCHECK(!context.IsEmpty());
+  bool success = node::InitializeContext(renderer_context);
```

Comment:
/nit

Can we just `DCHECK(node::InitializeContext(renderer_context))`.

The thing inside the `DHCHECK` is what gets printed out and `check failed "success"` isn't as helpful as `check failed "node::InitializeContext(renderer_context)"`

Same with the other one